installer.sh

```diff
@@ -16,7 +16,7 @@ LATEST_VERSION="${LATEST_VERSION##*v}"
 
 PLATFORM="${PLATFORM:-}"
 TMP_DIR=$(mktemp -d)
-OUT_DIR="${OUT_DIR:-/usr/local/bin}"
+OUT_DIR="${OUT_DIR:-$HOME/.metatype/bin}"
```

Comment:
Might I recommend the XDG directories instead? The binaries would go into `~/.local/bin` instead. Any other state would go to `~/.local/` or  `~/.config`. @zifeo wdyt?

Reply:
XDG convention may indeed be even better - but might be hard to check if on path already (not standard on Mac)?

Reply:
XDG bin does seem to be ill defined on Mac. Let's keep with `.metatype` then.